### PR TITLE
add rospkg [ROS]

### PR DESCRIPTION
**Pull request by wolfv**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rospkg/recipe/LICENSE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Copyright <YEAR> <COPYRIGHT HOLDER>
```
Comment (xhochy) [inline recipes/rospkg/recipe/LICENSE]: Is this the acutal LICENSE file?

File: recipes/rospkg/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  entry_points:
+    - rosversion = rospkg.rosversion:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - catkin_pkg
```
Comment (xhochy) [inline recipes/rospkg/recipe/meta.yaml]: The following two dependencies don't seem to be required.

File: recipes/rospkg/recipe/LICENSE (hunk 1 above)
Comment (wolfv) [reply recipes/rospkg/recipe/LICENSE]: The problem is that they don't have any license file. The license is only indicated in the setup.py. should I select that as license file?

Comment (wolfv) [reply recipes/rospkg/recipe/LICENSE]: Ok, I copied the header of one of the files of rospkg now. Seems more appropriate.

File: recipes/rospkg/recipe/meta.yaml (hunk 2 above)
Comment (wolfv) [reply recipes/rospkg/recipe/meta.yaml]: removed them from host. let's see if it builds fine.

Comment (wolfv) [reply recipes/rospkg/recipe/meta.yaml]: It did :+1:

Comment (xhochy) [reply recipes/rospkg/recipe/meta.yaml]: `host` is basically what `setup.py` requires. `conda skeleton pypi` sadly cannot infer this.

**Comment by wolfv** [comment]
thanks @xhochy! Is there any way that I can trigger CI on the feedstocks so I can push some more recipes later today? It doesn't seem to activate for me:

https://github.com/conda-forge/empy-feedstock
https://github.com/conda-forge/log4cxx-feedstock
https://github.com/conda-forge/sbcl-feedstock
https://github.com/conda-forge/tinyxml-feedstock

**Comment by xhochy** [comment]
@wolfv Try pushing an empty commit, otherwise you will need to reach out to `@conda-forge/core`.

**Comment by wolfv** [comment]
empty commits worked, thanks!

**Pull request merged**